controllers/providers/aws/ssm.go

```diff
@@ -60,8 +65,16 @@ func GetAwsSsmClient(region string, cacheCfg *cache.Config, maxRetries int, coll
 }
 
 func (w *AwsWorker) GetEksLatestAmi(OSFamily string, arch string, kubernetesVersion string) (string, error) {
+	return w.GetEksSSMAmi(OSFamily, arch, kubernetesVersion, LatestIdentifiers[OSFamily])
+}
+
+func (w *AwsWorker) GetEksSSMAmi(OSFamily string, arch string, kubernetesVersion string, ssmId string) (string, error) {
+	var inputString = aws.String(fmt.Sprintf(EksAmis[OSFamily][arch], kubernetesVersion, ssmId))
+	if OSFamily == "windows" {
+		inputString = aws.String(fmt.Sprintf(EksAmis[OSFamily][arch], ssmId, kubernetesVersion))
```

Comment:
Do we validate the annotation value OSFamily / Arch upstream prior to this? Can you make sure that invalid annotation value to dont result in exception due to map lookup?